src/diffusers/models/autoencoder_kl.py

```diff
@@ -56,6 +56,13 @@ class AutoencoderKL(ModelMixin, ConfigMixin):
         act_fn (`str`, *optional*, defaults to `"silu"`): The activation function to use.
         latent_channels (`int`, *optional*, defaults to `4`): Number of channels in the latent space.
         sample_size (`int`, *optional*, defaults to `32`): TODO
+        scaling_factor (`float`, *optional*, defaults to `0.18215`):
```

Comment:
very nice docstring! 
```suggestion
        scaling_factor (`float`, *optional*, defaults to 0.18215):
```